logging/jboss/src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+#When you set the logging level below the minimum logging level, you must adjust the minimum logging level as well.
+#Otherwise, the value of minimum logging level overrides the logging level.
+# DOC:
+# https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/1.11/html-single/configuring_logging_with_quarkus/index#ref-example-logging-configuration_quarkus-configuring-logging
+# https://quarkus.io/guides/logging
+# https://quarkus.io/guides/all-config#quarkus-core_quarkus.log.min-level
+
+#quarkus.log.min-level=DEBUG
```

Comment:
To make things more obvious, I would suggest to either set the `min-level` explicitly (i.e. uncomment this line), or mention in the comment that `DEBUG` is the default.